web/app/scripts/views/map/layers-controller.js

```diff
@@ -213,87 +204,103 @@
                 return;
             }
 
-            // remove overlays if already added
-            if (ctl.overlays) {
-                angular.forEach(ctl.overlays, function(overlay) {
-                    ctl.map.removeLayer(overlay);
-                });
-            }
-
-            var defaultLayerOptions = {attribution: 'PRS', detectRetina: true};
-
-            // Event record points. Use 'ALL' or record type UUID to filter layer
-            var recordsLayerOptions = angular.extend(defaultLayerOptions, {zIndex: 3});
-            var recordsLayer = new L.tileLayer(ctl.getFilteredRecordUrl(recordsUrl), recordsLayerOptions);
-
-            // layer with heatmap of events
-            var heatmapOptions = angular.extend(defaultLayerOptions, {zIndex: 4});
-            var heatmapLayer = new L.tileLayer(ctl.getFilteredRecordUrl(heatmapUrl), heatmapOptions);
+            $q.all([TileUrlService.recTilesUrl(ctl.recordType),
+                    TileUrlService.recUtfGridTilesUrl(ctl.recordType),
+                    TileUrlService.recHeatmapUrl(ctl.recordType)]).then(function(tileUrls) {
+                var baseRecordsUrl = tileUrls[0];
+                var baseUtfGridUrl = tileUrls[1];
+                var baseHeatmapUrl = tileUrls[2];
+                var defaultLayerOptions = {attribution: 'PRS', detectRetina: true};
```

Comment:
Done; I only changed this to false for the base layer, so boundaries and events should still come in at high res.